### DISCUSSION TITLE
How to allow the popular gps file extensions in the GitHub discussions

### DISCUSSION THREAD

**Discussion by 0709wiwiwi** [devemux86/cruiser#231] [Q&A] [answered]
**Inserted as a new entry in "general" so that this notification is not lost sight of.**

Particularly annoying that the github discussions do not allow more (gps) file extensions in uploads. 
Each time this way, discussion time is lost before new users catch on to this somewhat discouraging flaw. 
 
**If one has a best advice to report this annoyance and resolve it their help will be gladly accepted.**

> You can attach kurviger files if you zip them or change their extension to a known one, like txt.

The commonly used gps extensions are unfortunately not accepted for direct import in the github discussions.
 
Add your votes and thumbs up here:  [Allow additional file extensions in github discussions](https://github.com/orgs/community/discussions/58245)

Possibly more Cruiser users could add a thumbs up, or even post an extra comment to report this annoying flaw. 
I don't know if I requested this in the most optimal place in Github.  However, it still remains without any response.

**Comment by MotoUKRider** [comment]
It's annoying, but understandable - "odd" file extensions *may* be seen as a potential threat despite containing a perfectly innocent series of boring Waypoints!

File extensions aren't enabled by default in Windoze, are they?  The "Hide Extensions for known file types" was an option I always used to turn-off very quicky in v7.

**Reply by 0709wiwiwi** [reply, marked as answer]
;-)  Put .txt or .zip after it and thus it is no longer harmful ? I am from the generation where military service was still obligatory. To protect the barrel from grains of sand you could put a very flexible rubber protection over it, but it did not make the bullets more harmless.
